quadratichq/quadratic
# fix: redo command for windows

Fixes #384 

Expectation ([per Andrew](https://quadratichq.slack.com/archives/C03VC0Q1MPA/p1680293627232329?thread_ts=1680232285.716439&cid=C03VC0Q1MPA)) for Windows users:

- Undo: `ctrl` + `Z`
- Redo: `ctrl` + `shift` + `Z` OR `ctrl` + `Y`

Also fixes a bug where "File" -> "Redo" had the arrow pointed in the wrong direction

Before:

<img width="181" alt="CleanShot 2023-03-31 at 14 50 35@2x" src="https://user-images.githubusercontent.com/1316441/229227793-dad66a6e-05e3-473d-9402-b50d5c9f4ee1.png">

After:

<img width="180" alt="CleanShot 2023-03-31 at 14 49 58@2x" src="https://user-images.githubusercontent.com/1316441/229227806-dabf6d97-06a6-4f4d-86be-1ba6c7dfc3ec.png">


**To test:**

- Ensure the keyboard commands work
- Ensure Redo shows up in File -> Edit -> Redo with the correct shortcuts
- Ensure Redo shows up in the command palette with the correct shortcuts

---

(Note: doing this work only reinforces [the need to centralize shortcuts in one place in the code](https://github.com/quadratichq/quadratic/issues/312))
